bot/storage/diskstorage.py

```diff
@@ -0,0 +1,30 @@
+import os
+import json
+import logging
+import datetime
+
+import storage
+
+class DiskStorage(storage.Storage):
+    def __init__(self):
+        super(storage.Storage, self).__init__()
```

Comment:
Good updates! While I'm not too hot on using the environment in the constructor vs passing it in, it will just add unneeded complexity to fix it so :+1:

Reply:
I agree but I didn't want to refactor the application any more than necessary, so I adopted env dependency.